BinariesAPI.md

```diff
@@ -0,0 +1,107 @@
+# JULEP BinariesAPI
+
+- **Title:** A structured HTTP API for getting Julia binaries
+- **Authors:** Eric Davies <<iamed2@gmail.com>>
+- **Created:** June 16, 2017
+- **Status:** proposed
+
+## Abstract
+
+The URLs for fetching Julia binaries of various versions have changed a few times across
+releases.
+The latest change necessitated PRs to multiple repos to change AppVeyor scripts.
+This Julep proposes a level of abstraction over the Julia binary URLs which is decoupled
+from the storage service used for the binaries.
```

Comment:
But not quite. There are still two domains at play. And AFAICT it still relies on a file structure (with one copy of each file added for each alias).